runtime/vm/createramclass.cpp

```diff
@@ -2079,6 +2064,22 @@ loadFlattenableFieldValueClasses(J9VMThread *currentThread, J9ClassLoader *class
 	if (NULL != superClazz) {
 		*valueTypeFlags |= (J9ClassHasReferences & superClazz->classFlags);
 	}
+
```

Comment:
Could you also update the doxygen comment of this function to mention the new behaviour ?

Reply:
Updated.